apps/web/src/app/(dashboard)/documents/page.tsx

```diff
@@ -110,21 +98,30 @@ export default async function DocumentsPage({ searchParams = {} }: DocumentsPage
                 </span>
               </Link>
             </TabsTrigger>
+
+            <TabsTrigger className="min-w-[60px]" value={InternalDocumentStatus.DRAFT} asChild>
+              <Link href={getTabHref(InternalDocumentStatus.DRAFT)}>
+                <DocumentStatus status={InternalDocumentStatus.DRAFT} />
+
+                <span className="ml-1 hidden opacity-50 md:inline-block">
+                  {Math.min(stats.DRAFT, 99)}
+                </span>
+              </Link>
+            </TabsTrigger>
+
+            <TabsTrigger className="min-w-[60px]" value="ALL" asChild>
+              <Link href={getTabHref('ALL')}>All</Link>
+            </TabsTrigger>
           </TabsList>
         </Tabs>
 
         <div className="flex flex-1 flex-wrap items-center justify-between gap-x-2 gap-y-4">
           <PeriodSelector />
-
-          <Button>
-            <Plus className="-ml-1 mr-2 h-5 w-5" />
-            Add Document
-          </Button>
         </div>
       </div>
 
       <div className="mt-8">
-        {/* If we're viewing all documents for all time and there's nuffin we should should an add document component instead */}
+        {/* If we're viewing all documents for all time and there's nuffin we should should show an add document component instead */}
         {isNoResults ? (
           <DocumentDropzone className="min-h-[60vh] md:min-h-[40vh]" />
```

Comment:
Wouldn't make sense to show this twice now would it 🤔

Reply:
Interesting point. Considering the massive CTA above, another button (even if small) seems redundant.

Reply:
Removed it